contracts/btc-staking/src/contract.rs

```diff
@@ -219,40 +214,13 @@ pub fn sudo(
 }
 
 fn handle_begin_block(deps: &mut DepsMut, env: Env) -> Result<Response<BabylonMsg>, ContractError> {
-    // Index BTC height at the current height
-    index_btc_height(deps, env.block.height)?;
-
```

Comment:
This seems necessary for determining whether BTC delegations are expired due to timelock?

Reply:
Yes, you're right. But we're currently gathering that info from Babylon. So, it's not needed at this point.

Reply:
Have we decided to let Babylon side or consumer side to determine expiration of BTC delegations? I remember we are trying to figure out an algorithm that is executed by consumer side

Reply:
Not sure. I think there's an issue for making Babylon forward those undelegations to the Consumers. Let me check.

Reply:
I cannot find the issue, but I remember we discussed this at some point. @gusin13?

Reply:
What about creating an issue to re-introduce this indexing at some point when it's needed?

Reply:
Here: #66.